src/core/printer.ts

```
@@ -121,6 +132,7 @@ export class Printer {
 
     if (this.shouldPrint()) {
       spinner.start();
+      this.activeSpinner = spinner;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)